app/data/census_household.json

```diff
@@ -1429,12 +1429,17 @@
                                                 {
                                                     "label": "Yes, an address outside the UK (please specify)",
                                                     "value": "Other",
-                                                    "other": {
-                                                        "label": "Please enter country"
-                                                    }
+                                                    "child_answer_id": "another-address-answer-other"
```

Comment:
How else do you know if an answer has a parent, from an answers json? We could iterate the entire schema and build a map, but seems like quite an overhead when inspecting each answer.